index.html

```diff
@@ -10,6 +10,7 @@
 </head>
 
 <body>
+
     <div class="container">
         <div class="col-md-12">
             <h1 id="title">Survey Form</h1>
```

Comment:
Great Job! 👍 on using the hx tag for the title and using h1 once in your code, it makes your project more semantic.